### DISCUSSION TITLE
before_build中拷贝一个文件夹作为add_includedirs，会偶发一个warning

### DISCUSSION THREAD

**Discussion by chanchann** [xmake-io/xmake#3210] [Q&A]
https://github.com/sogou/workflow/blob/49709891eb6e9edaab9df4b81b9072ce0a2f6b53/src/xmake.lua#L24

我在before_build中，会先将头文件拷贝到`_include`文件夹中

https://github.com/sogou/workflow/blob/49709891eb6e9edaab9df4b81b9072ce0a2f6b53/xmake.lua#L72

我将这个`_include`文件夹设置为`includedirs`

编译偶尔会出现这个warning，但是不影响编译，请问下大佬怎么解决这个warning🙏

```
warning: target(nameservice).add_includedirs("/Users/chanchan/Documents/pro/workflow/_include") path not found at ./xmake.lua:72
```

**Comment by waruqi** [comment]
是你在 before_build 阶段后期创建的，有这个warning 正常，你提前创建好这个目录 就不会报了。。或者在 on_load 阶段就创建好

**Comment by chanchann** [comment]
非常感谢！
